pkg/blobstore/grpcservers/byte_stream_server_test.go

```diff
@@ -15,13 +16,13 @@ import (
 	"github.com/buildbarn/bb-storage/pkg/testutil"
 	"github.com/stretchr/testify/require"
 
+	"github.com/klauspost/compress/zstd"
+	"go.uber.org/mock/gomock"
```

Comment:
Please keep imports grouped by second level domain:

- github.com
- golang.org / google.golang.org
- go.uber.org

Reply:
ack. What about the internal ones? Do those stay grouped? I notice stretcher testify is grouped with bb-storage

We should probably come up with some auto-formatter to do this for us. I've also seen <stdlib><external><generated>